core/src/test/scala/org/apache/spark/sql/delta/ActionSerializerSuite.scala

```diff
@@ -213,15 +213,22 @@ class ActionSerializerSuite extends QueryTest with SharedSparkSession {
     expectedJson = """{"cdc":{"path":"part=p1/f1","partitionValues":{"x":null},""" +
       """"size":10,"dataChange":false}}""".stripMargin)
 
-  testActionSerDe(
-    "Metadata (with all defaults) - json serialization/deserialization",
-    Metadata(createdTime = Some(2222)),
-    expectedJson = """{"metaData":{"id":"testId","format":{"provider":"parquet",""" +
-    """"options":{}},"partitionColumns":[],"configuration":{},"createdTime":2222}}""")
+  {
+    // We want this metadata to be lazy so it is instantiated after `SparkFunSuite::beforeAll`.
+    // This will ensure that `Utils.isTesting` returns true and that its id is set to 'testId'.
```

Comment:
where is `Utils.isTesting` used? can you just point me to that?

Reply:
https://github.com/delta-io/delta/issues/1283#issuecomment-1195663498

In the creation of Metadata id

Reply:
https://github.com/delta-io/delta/issues/1283#issuecomment-1195663498

In the creation of Metadata id